starboard/tools/testing/test_runner.py

```diff
@@ -786,7 +786,7 @@ def BuildAllTargets(self, ninja_flags):
 
       # The loader is not built with the same platform configuration as our
       # tests so we need to build it separately.
-      if self.loader_platform:
+      if self.loader_platform and self.loader_platform.find("rdk") == -1:
```

Comment:
Same comment about platform specific code here

Reply:
Removed the platform specific check from test_runner.